python/llm/dev/benchmark/all-in-one/run-igpu.sh

```diff
@@ -0,0 +1,6 @@
+source /opt/intel/oneapi/setvars.sh
+ 
+export USE_XETLA=OFF
+export SYCL_CACHE_PERSISTENT=1
```

Comment:
Seems like we should have
```
export SYCL_CACHE_PERSISTENT=1
export BIGDL_LLM_XMX_DISABLED=1
```

![image](https://github.com/intel-analytics/ipex-llm/assets/54161268/1b79cb11-504a-4335-981f-c87ba7465e42)